### ISSUE TITLE
Training RCAN_BIX3 and RCAN_BIX4

### ISSUE THREAD

**Issue by dongheehand** [yulunzhang/RCAN#62]
I used the pre-trained RCAN_BIX2 to train RCAB_BIX3 or RCAB_BIX4, as written in your TrainRCAN_scripts.sh .
However, the training loss is too large. If I use pre-trained RCAB_BIX2 to train RCAB_BIX3 or RCAB_BIX4, I think the initial learning rate is not 1e-4.

I am wondering the initial learning rate used to train RCAB_BIX3 or RCAB_BIX4 if pre-trained RCAB_BIX2 is used.

**Comment by yulunzhang** [comment]
Hi,

When we train X3/X4 models based on the pretrained X2 model, we set the initial learning rate as 1e-4.

**Comment by Nkriskeeic** [comment]
Hi, I have the same trouble as @dongheehand  has.
Does it decrease as learning progresses ?
Or, I would appreciate it if you could provide the learning curve for training loss.

**Comment by DamperHa** [comment]
I trained this model from scratch, but at the beginning of the training, the training loss was very large and did not come down. Do you know how to solve it?

**Comment by LcenArthas** [comment]
I had the same problem, and at a certain point in the training, loss became very large.

**Comment by dunkle** [comment]
> I had the same problem, and at a certain point in the training, loss became very large.

have u solved it, could u give me some advice?

**Comment by DamperHa** [comment]
Hello, I can't remember it clearly because it has been a little long.&nbsp;It seems that the author provided several loss functions and training strategies in the code. You can try to modify them. I made some changes in this, and the model&nbsp;can be converged in my dataset.&nbsp;Here are a few parameters I used to converge:


&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;"type":&nbsp;"ADAM",
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;"learning_rate":&nbsp;1e-4,
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;"weight_decay":&nbsp;0,
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;"lr_scheme":&nbsp;"MultiStepLR",
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;"lr_steps":&nbsp;[100,&nbsp;200,&nbsp;300,&nbsp;400],
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;"lr_gamma":&nbsp;0.5,
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;"loss_type":&nbsp;"l1",
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;"manual_seed":&nbsp;0,
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;"num_epochs":&nbsp;500,


Of course, The parameter given by the author can also be convergent, but it needs to be modified. But I am so sorry, i can't remember exactly how to modify.




Good luck for you!!!










------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "yulunzhang/RCAN"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2020年9月21日(星期一) 下午2:50
收件人:&nbsp;"yulunzhang/RCAN"<RCAN@noreply.github.com&gt;;
抄送:&nbsp;"范志豪"<1186962425@qq.com&gt;;"Comment"<comment@noreply.github.com&gt;;
主题:&nbsp;Re: [yulunzhang/RCAN] Training RCAN_BIX3 and RCAN_BIX4 (#62)





  
I had the same problem, and at a certain point in the training, loss became very large.
  
have u solved it, could u give me some advice?
